InstructorFeedbackPageuUiTest unstable at line 866.

`java.lang.AssertionError: expected:<10> but was:<1>`
This seems to be caused by a laggy / missed dropdown selection from line 858.
May be Selenium's fault; the dropdown opens and the value is selected, but it seems to not commit the result (in human terms, it clicks on the dropdown, hovers over the value, and stops before clicking on it).

Works fine when running it on debug and ensuring the values are filled correctly.

--- a/src/test/java/teammates/test/pageobjects/InstructorFeedbacksPage.java
+++ b/src/test/java/teammates/test/pageobjects/InstructorFeedbacksPage.java
@@ -294,11 +294,13 @@ public class InstructorFeedbacksPage extends AppPage {
         fillTimeValueIfNotNull(Const.ParamsNames.FEEDBACK_SESSION_PUBLISHDATE, publishTime, publishTimeDropdown, js);
     }
     
-    public void fillTimeValueIfNotNull(String timeId, Date timeValue, WebElement timeDropdown, JavascriptExecutor js) {
-        if (timeValue != null) {
-            js.executeScript("$('#" + timeId + "')[0].value='" + TimeHelper.formatDate(timeValue) + "';");
-            selectDropdownByVisibleValue(timeDropdown,
-                                         TimeHelper.convertToDisplayValueInTimeDropDown(timeValue));
+    public void fillTimeValueIfNotNull(String dateId, Date datetimeValue, WebElement timeDropdown, JavascriptExecutor js) {
+        if (datetimeValue != null) {
+            js.executeScript("$('#" + dateId + "').val('" + TimeHelper.formatDate(datetimeValue) + "');");
+            
+            String timeDropdownId = timeDropdown.getAttribute("id");
+            String timeDropdownVal = TimeHelper.convertToOptionValueInTimeDropDown(datetimeValue);
+            js.executeScript("$('#" + timeDropdownId + "').val(" + timeDropdownVal + ")");
         }
     }
     

